### PR TITLE
Updated to 3.0.2 API

### PR DESCRIPTION
**Pull request by RachelTucker**

### CODE REVIEW

File: Ds3/Calls/GetStorageDomainsSpectraS3Request.cs [hunk 1]
```diff
@@ -233,6 +240,18 @@ public GetStorageDomainsSpectraS3Request WithPageStartMarker(string pageStartMar
             }
             return this;
         }
+        public GetStorageDomainsSpectraS3Request WithSecureMediaAllocation(bool? secureMediaAllocation)
+        {
+            this._secureMediaAllocation = secureMediaAllocation;
+            if (secureMediaAllocation != null) {
```
Comment (shabtaisharon) [inline Ds3/Calls/GetStorageDomainsSpectraS3Request.cs]: the correct way to check if a Nullable object is not null is by doing "secureMediaAllocation.HasValue"

File: Ds3/Calls/GetStorageDomainsSpectraS3Request.cs [hunk 2]
```diff
@@ -233,6 +240,18 @@ public GetStorageDomainsSpectraS3Request WithPageStartMarker(string pageStartMar
...
+                this.QueryParams.Add("secure_media_allocation", secureMediaAllocation.ToString());
```
Comment (shabtaisharon) [inline Ds3/Calls/GetStorageDomainsSpectraS3Request.cs]: and than to get the value "secureMediaAllocation.Value"

File: Ds3/Calls/ModifyStorageDomainSpectraS3Request.cs [hunk 3]
```diff
@@ -211,6 +218,18 @@ public ModifyStorageDomainSpectraS3Request WithName(string name)
             }
             return this;
         }
+        public ModifyStorageDomainSpectraS3Request WithSecureMediaAllocation(bool? secureMediaAllocation)
+        {
+            this._secureMediaAllocation = secureMediaAllocation;
+            if (secureMediaAllocation != null) {
```
Comment (shabtaisharon) [inline Ds3/Calls/ModifyStorageDomainSpectraS3Request.cs]: same Nullable style change

File: Ds3/Calls/ModifyStorageDomainSpectraS3Request.cs [hunk 4]
```diff
@@ -211,6 +218,18 @@ public ModifyStorageDomainSpectraS3Request WithName(string name)
...
+                this.QueryParams.Add("secure_media_allocation", secureMediaAllocation.ToString());
```
Comment (shabtaisharon) [inline Ds3/Calls/ModifyStorageDomainSpectraS3Request.cs]: same

File: Ds3/Calls/PutStorageDomainSpectraS3Request.cs [hunk 5]
```diff
@@ -192,6 +199,18 @@ public PutStorageDomainSpectraS3Request WithMediaEjectionAllowed(bool? mediaEjec
             }
             return this;
         }
+        public PutStorageDomainSpectraS3Request WithSecureMediaAllocation(bool? secureMediaAllocation)
+        {
+            this._secureMediaAllocation = secureMediaAllocation;
+            if (secureMediaAllocation != null) {
```
Comment (shabtaisharon) [inline Ds3/Calls/PutStorageDomainSpectraS3Request.cs]: same Nullable style change

File: Ds3/Calls/PutStorageDomainSpectraS3Request.cs [hunk 6]
```diff
@@ -192,6 +199,18 @@ public PutStorageDomainSpectraS3Request WithMediaEjectionAllowed(bool? mediaEjec
...
+                this.QueryParams.Add("secure_media_allocation", secureMediaAllocation.ToString());
```
Comment (shabtaisharon) [inline Ds3/Calls/PutStorageDomainSpectraS3Request.cs]: same

File: Ds3/Calls/GetStorageDomainsSpectraS3Request.cs (hunk 1 above)
Comment (rpmoore) [inline Ds3/Calls/GetStorageDomainsSpectraS3Request.cs]: This is probably a single change in the autogen

**Pull request closed without merging**